modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -95,6 +95,11 @@
 	<td>//div[@class='dropdown-menu show']//button[contains(text(),'Edit')]</td>
 	<td></td>
```

Comment:
FYI, this path we can use in both endpoint and schema, I think we can modify the name to 'DROPDOWN_TOGGLE'

Reply:
I will forward this since you already make this update in your pull request, thanks.